pysd/py_backend/builder.py

```diff
@@ -1006,8 +1043,28 @@ def build_function_call(function_def, user_arguments):
 
     """
     if isinstance(function_def, str):
+        if function_def[:10] == "functions.":
```

Comment:
is this the right way to do this? wouldn't we rather just have the `function_def` string not have the leading `functions.` wherever its coming from?

Reply:
In order to identify when we have to import a new method from PySD in the model library I just used this, however, we could make different ways. But I take this option so I didn't need to make changes in the vensim2py or xmile2py files.

Reply:
I have an idea to make this implementation cleaner and more solid. Give me half an hour.

Reply:
@JamesPHoughton I made the changes in xmile2py SMILE2Py and builder to make this more solid. You can check the changes. See that I added the "module" information to the functions and make all of them use `builder.build_function_call`, which will update the importing methods list.